src/services/Redis.ts

```diff
@@ -1,63 +1,110 @@
 import { ClientOpts, createClient, RedisClient } from 'redis';
-import { promisify } from 'util';
+import { EventEmitter } from 'events';
 import Expire from '../types/Expire';
 import Service from '../types/Service';
 
+enum methodLabels {
```

Comment:
I am not sure if there is industry standard for enums in TS. But I have always seen only two implementations -> https://www.typescriptlang.org/docs/handbook/enums.html
Same method labels is uber generic.
Maybe:
```
enum RedisActions {
    Get = 'get',
    .......
}
```

Reply:
👍 Done